gapir/cc/memory_manager.h

```diff
@@ -24,8 +24,12 @@
 #include <utility>
 #include <vector>
 
+#include "memory_allocator.h"
+
 namespace gapir {
 
+class MemoryAllocator;
+
 // Memory manager class for managing the memory used by the replay system.
 // Outside of this class there shouldn't be any significant heap allocation in
```

Comment:
This comment may become out-of-sync, since the big heap allocation is actually done in MemoryAllocator ?

Also below, is the layout comment still relevant?